src/Button.php

```diff
@@ -27,9 +27,8 @@
 final class Button extends \Yiisoft\Widget\Widget
 {
     private const NAME = 'btn';
-    private bool $active = false;
     private array $attributes = [];
-    private ButtonVariant $buttonVariant = ButtonVariant::SECONDARY;
+    private ButtonVariant|null $buttonVariant = null;
```

Comment:
https://getbootstrap.com/docs/5.3/components/buttons/#base-class For the button without variant.

Reply:
Suggest keep default value, but add ability to set `null`.

Reply:
I would like to keep `null` as the default value, the button has no default variant, according to the documentation, and by having `null` value, the code is kept simple since i don't need to do any checking and there is no need to add `psalm-supress` for the property with no value and not initialized in the constructor.